tests/python/gaia-ui-tests/gaiatest/resources/sample_websites/darkness/pages/style-demo.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

```

Comment:
Actually these being a ready-made website template, my plan is to expand my imagecompare test cases by populating them with contents in the future.  (one example is XHTML, and the other is reactive HTML5 template)  This reminds me, I should ping ato about the status of the pinch() method in marionette, since they are still broken at the moment.